salt/modules/zcbuildout.py

```diff
@@ -974,7 +960,7 @@ def buildout(
         salt '*' buildout.buildout /srv/mybuildout
     """
     LOG.info(
-        "Running buildout in {0} ({1})".format(directory, config)
+        "Running buildout in {} ({})".format(directory, config)
     )  # pylint: disable=str-format-in-logging
```

Comment:
```suggestion
        "Running buildout in %s (%s)", directory, config
    )
```

Reply:
LOG is not a normal logger instance, We'll need to resolve this in a different PR.